### DISCUSSION TITLE
Option to update Client name?

### DISCUSSION THREAD

**Discussion by diogo1995santo** [astro-panda/Clockify.Net#43] [Q&A] [answered]
Hello!

Are there currently any plans to add an option to update a Client name through this client?

**Comment by Morasiu** [comment]
Hey.

Start an Issue with that feature request.

Maybe someone will work on this. I am not using that library anymore so I am not keen to work on it, but I will happily accept some PRs.

**Comment by Psypher9** [comment, marked as answer]
Looks like this has already been implemented and is accessible from `_clockifyClient.UpdateClientNameAsync()`!
